packages/wonder-blocks-modal/components/one-pane-dialog/one-pane-dialog.js

```diff
@@ -0,0 +1,152 @@
+
+    /**
+     * When set, provides a component that can render content above the top of the modal;
+     * when not set, no additional content is shown above the modal.
+     * This prop is passed down to the ModalDialog.
+     *
+     * NOTE: Devs can customize this content by rendering the component assigned to this prop with custom styles,
+     * such as by wrapping it in a View.
+     */
+    above?: React.Node,
+
+    /**
+     * When set, provides a component that will render content below the bottom of the modal;
+     * when not set, no additional content is shown below the modal.
+     * This prop is passed down to the ModalDialog.
+     *
+     * NOTE: Devs can customize this content by rendering the component assigned to this prop with custom styles,
+     * such as by wrapping it in a View.
+     */
+    below?: React.Node,
+
+    /**
+     * Optional custom styles.
+     */
+    style?: StyleType,
+|};
+
+/**
+ * This is the standard layout for most straightforward modal experiences.
+ *
+ * The ModalHeader is required, but the ModalFooter is optional.
+ * The content of the dialog itself is fully customizable, but the left/right/top/bottom padding is fixed.
+ */
+export default class OnePaneDialog extends React.Component<Props> {
+    renderCloseButton() {
+        const {onClose} = this.props;
+
+        return (
+            <MediaLayout styleSheets={styleSheets}>
+                {({styles}) => (
+                    <CloseButton
+                        light={false}
+                        onClick={onClose}
+                        style={styles.closeButton}
+                    />
+                )}
+            </MediaLayout>
+        );
+    }
+
+    render() {
+        const {
+            onClose,
+            title,
+            subtitle,
+            footer,
+            content,
+            above,
+            below,
+            style,
+        } = this.props;
+
+        return (
+            <MediaLayout styleSheets={styleSheets}>
+                {({styles}) => (
+                    <ModalDialog
+                        style={[styles.wrapper, style]}
+                        above={above}
+                        below={below}
+                    >
+                        <ModalPanel
+                            onClose={onClose}
+                            titleBar={
+                                <Toolbar
+                                    title={title}
+                                    subtitle={subtitle}
+                                    color="light"
+                                    rightContent={this.renderCloseButton()}
+                                />
+                            }
+                            content={content}
+                            footer={footer}
+                        />
+                    </ModalDialog>
+                )}
+            </MediaLayout>
+        );
+    }
+}
+
+const styleSheets = {
+    all: StyleSheet.create({}),
```

Comment:
TODO: Leaving this empty objects as I might need them after applying design specs into the implementation. I can remove this right now if you consider them unnecessary :)